activate.py

```diff
@@ -5,25 +5,46 @@
 import argparse
 
 
-class KeysSubgraph:
+class Keysmanager:
     url = None
+    validators = None
 
     def __init__(self, url):
         self.url = url
 
-    def get_validators(self, address: str):
+    def get_validators(self, time, address):
         query = """ {
-  validators(where: {nodeOperator: "address"}) {
-    id
-    nodeOperator
-    publicKey
-    signature
-    status
-  }
-        }
-          """.replace("address", address)
+            validators(
+                first: 1000
+                where: {timestamp_gt: "time_start", nodeOperator: "address"}
```

Comment:
is pagination here done with timestamp only? What happens when there is timestamp clashes here?
From the docs it looks like one uses `skip` instead, which kinda makes sense and makes it timestamp independent.
https://thegraph.com/docs/en/developer/developer-faq/#22-is-there-a-limit-to-how-many-objects-the-graph-can-return-per-query

Reply:
implemented skip